_posts/2019-07-12-service-design-anti-patterns.md

```diff
@@ -0,0 +1,28 @@
+
+# Digital service design
+Whilst we’re living in a time of the highest internet access and digital skills in history, there will always be someone (likely more than one) who can’t use your digital service. This might be related to a permanent barrier – like a disability or health condition – or something more circumstantial – like waiting for your broadband to be switched on. Digital-only models are attractive to service providers as they create the potential for greater operational and administrative savings whilst providing increased availability to the end-user. However, they can often exclude some people from using the service and lead them to look elsewhere. Similarly, when redesigning an existing service, if you don’t extend your scope to the offline touch-points and support processes you’ll quickly create a disconnected or inconsistent experience.
+
+Understanding your users’ behaviour and designing a service with the appropriate touch-points and channels in place ensures everyone receives the same great level of service.
+
+# Up-front process
+Service Design is not making blueprints. It’s not about trying to document a near-perfect vision of the future and then handing it over for someone else to deliver. The output of Service Design should be – as with all other fields of design – the thing you’re designing: the service. The longer you spend designing upfront the longer it takes to get real feedback, and there’s a real possibility [you may get stuck and never actually deliver anything](https://medium.com/hippo-digital/service-design-how-not-to-get-stuck-in-a-never-ending-discovery-3536d225087c). Detaching design and implementation is rarely effective, particularly when the thing you’re designing has so many moving parts. A more successful approach is to cross the river by feeling the stones: breaking down the challenge into small chunks, designing incrementally and adapting quickly to feedback. It’s also worth emphasising that Service Design doesn’t necessarily have to be done by Service Designers (possibly an anti-pattern, itself). Whilst these individuals can often help give form to an initial vision and coach less familiar team, the goal should be for the team to take responsibility for continuously optimising the service and designing it around user needs.
```

Comment:
Suggest adding a word: _Service Design is not **just** making blueprints_. Suggest adding a comma after 'upfront': _The longer you spend designing upfront, the longer it takes to get real feedback_. Suggest adding ",as popularised by Deng Xiaoping" after _cross the river by feeling the stones_.